src/main/java/teammates/common/util/Sanitizer.java

```diff
@@ -186,8 +194,8 @@ public static String sanitizeForSearch(String str){
      * We follow the definition described by RFC 4180:<br>
      * {@link http://tools.ietf.org/html/rfc4180}
      */
-    public static String sanitizeForCsv(String str) {
-        return "\"" + str.replace("\"", "\"\"") + "\"";
+    public static String sanitizeForCsv(String str){ 
+        return "\"" + str.replace("\"", "\"\"") + "\""; // in CSV, " is escaped as ""
```

Comment:
I don't think the comment is necessary, if someone were to come and modify this method they'd probably research more about sanitizing for CSV and know that

If you think it's an obscure thing and should be commented, add it into the method comment and add it as a "note" rather than here